syntax = "proto3";
import "common_proto/common.proto";
import "common_proto/mysql_database_table.proto";
import "common_proto/node_common.proto";
option cc_generic_services = true;

message LoadAccountRequest
{
  string account = 1;
}

message LoadAccountResponse
{
	Tips error = 1;
  account_database account_player = 2;
}

message SaveAccountRequest
{
  string account = 1;
}

message SaveAccountResponse
{

}

message LoadPlayerRequest
{
  uint64 player_id = 1;
}

message LoadPlayerResponse
{
  uint64 player_id = 1;
}

message SavePlayerRequest
{
  uint64 player_id = 1;
}

message SavePlayerResponse
{
  uint64 player_id = 2;
}



service AccountDBService {
  rpc Load2Redis (LoadAccountRequest) returns (LoadAccountResponse);
  rpc Save2Redis (SaveAccountRequest) returns (SaveAccountResponse);
}

service PlayerDBService {
  rpc Load2Redis (LoadPlayerRequest) returns (LoadPlayerResponse);
  rpc Save2Redis (SavePlayerRequest) returns (SavePlayerResponse);
}
